test/rules/no-irregular-whitespaces/test.builder.ts

```diff
@@ -0,0 +1,28 @@
+import * as fs from 'fs';
```

Comment:
I might by mistaken, but I did not find a way to directly specify unicode representations of these invalid whitespaces in the tests ([eslint does that](https://github.com/eslint/eslint/blob/master/tests/lib/rules/no-irregular-whitespace.js#L34)).

Since we need to represent these whitespaces anyway, I made this small script to avoid any error when modifying these files. However, I have no idea how this can be included within the test pipeline.
The other solution is to `tsc ./test.builder.ts && node ./test.builder.js` manually but I don't find it satisfying ([ts-node](https://github.com/TypeStrong/ts-node) might be an alternative, not sure if we want to add a devDependency though).

Do you have any suggestion around that?